public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
```

Comment:
![Getting Started Section](https://user-images.githubusercontent.com/93039359/197803409-0f2f58f9-8cf3-4011-8fc1-fafc572521e1.PNG)

**[Optional]**

- [ ] You have made your readme great. 😊 It would be nice if your remove all the details from **Getting Started Section** from your readme which you can see in the image above ⬆️  Getting Started Section should contain the following commands ⬇️ 

```
git clone git@github.com:Fayob/maths-magician-app.git
npm install
npm start
```